packages/lab/src/portal/Portal.tsx

```diff
@@ -95,12 +62,9 @@ export const Portal = forwardRef<HTMLElement, PortalProps>(function Portal(
     return <>{children}</>;
   }
 
-  if (mounted && portalRef.current && children) {
-    return createPortal(
-      <SaltProvider>{children}</SaltProvider>,
-      portalRef.current
-    );
-  }
-
-  return null;
+  return (
+    <FloatingPortal root={container as HTMLElement} id={id}>
```

Comment:
What is FloatingPortal has more props added ?
There is no way to change defaults

[preserveTabOrder](https://floating-ui.com/docs/FloatingPortal#preservetaborder)`